src/python/pants/core/util_rules/adhoc_binaries_test.py

```diff
@@ -75,4 +75,4 @@ def test_docker_helper(rule_runner: RuleRunner):
         _PythonBuildStandaloneBinary,
         [_DownloadPythonBuildStandaloneBinaryRequest()],
     )
-    assert not pbs.path.startswith("/")
```

Comment:
Is it worth expanding this test to validate that it actually picks out a working interpreter? i.e. that the path is valid, not just that it's absolute.